apps/meteor/app/e2e/server/functions/handleSuggestedGroupKey.ts

```diff
@@ -21,9 +23,12 @@ export async function handleSuggestedGroupKey(
 		throw new Meteor.Error('error-no-suggested-key-available', 'No suggested key available', { method });
 	}
 
-	if (handle === 'accept') {
-		await Subscriptions.setGroupE2EKey(sub._id, suggestedKey);
-	}
+	const e2eKeyUpdateResult =
```

Comment:
this is changing the previous code behavior, where `unsetGroupE2ESuggestedKey` was always being called.